certif/app/services/locale.js

```diff
@@ -3,9 +3,15 @@ import config from 'pix-certif/config/environment';
 
 const { COOKIE_LOCALE_LIFESPAN_IN_SECONDS } = config.APP;
 
+export const DEFAULT_LOCALE = 'fr';
```

Comment:
```suggestion
export const DEFAULT_LOCALE=FRENCH_INTERNATIONAL_LOCALE;
```  
(à faire aussi sur les autres fichiers locale.js)